main.py

```diff
@@ -0,0 +1,63 @@
+import asyncio
+from examples.websocket_rfq import run
+
+# Configuration examples
+configuration = {
+    "DEV": {
+        "api_key": "9cfc9c3da4185d8f03d9f7962fec33d6",
+        "private_key": """-----BEGIN EC PRIVATE KEY-----
+MHcCAQEEIPQ/pSkixlGLBN5F6/hI7KsenK87FHgHG2dH1PQevUSxoAoGCCqGSM49
+AwEHoUQDQgAEzi3e/genFNJ7flOwUiuA1j5sv5wvsOhaeX2Wh3O1LOC5AHETJgoM
+tAZBmql9DA1hCwb4EIfa5OBdUEBjD3qQYw==
+-----END EC PRIVATE KEY-----""",
+        "ws_url": "wss://proxy.dev.ptlab.link"
+    },
+    "TEST": {
+        "api_key": "615ebf5163a08d3fc5526c04bb84c337",
+        "private_key": """-----BEGIN EC PRIVATE KEY-----
+MHcCAQEEIFu4h1DaVWX2IJLqOIFoxx81olmMFhtUXg0trJsSUKULoAoGCCqGSM49
+AwEHoUQDQgAEdeLG5CI63O4cBjsXIJc72ExJzciyBOY5nzH4XlT+CmNEp3Z1znwd
+Zkfv2Us/xCVFQ9wevr2ws1s/CxV8o+q3cQ==
+-----END EC PRIVATE KEY-----""",
+        "ws_url": "wss://proxy.test.ptlab.link"
+    },
+    "STAGING": {
+        "api_key": "c2554a11d0f27a9208278ff6649f70eb",
+        "private_key": """-----BEGIN EC PRIVATE KEY-----
+MHcCAQEEIORjLlCLWeYEa+ucy2QsM2A7Z7DA3WonZOnr/V6EvI6loAoGCCqGSM49
+AwEHoUQDQgAELYJenygoIyKZz80PUau/16dlYMWrrAa/FPakMWFJJ+5i49POmcB6
+yFMKWFghCx+145ZnWE9rJ7PksEh8szGIzg==
+-----END EC PRIVATE KEY-----""",
+        "ws_url": "wss://proxy.staging.ptlab.link"
+    },
+}
```

Comment:
@klugjo : I suggest to remove these configs from the code by placing to corresponding config files (e.g. JSON file(s) per environment: config.DEV.json,  config.TEST.json,  config.STAGING.json) and then reading from this code by command line parameter etc. The idea is to make code cleaner by getting rid off mutable values not related to the logic itself.

Reply:
(then clients may also use their own configs)

Reply:
Sure will do

Reply:
Done